programs/Makefile

```diff
@@ -331,6 +331,10 @@ test/benchmark$(EXEXT): test/benchmark.c $(DEP)
 	echo "  CC    test/benchmark.c"
 	$(CC) $(LOCAL_CFLAGS) $(CFLAGS) test/benchmark.c   $(LOCAL_LDFLAGS) $(LDFLAGS) -o $@
 
+test/cpp_dummy_build.cpp: test/generate_cpp_dummy_build.sh
```

Comment:
Why? This is a test file so I don't think it needs to ship in releases.

Reply:
No particular reason except it's a file that's generated, so I thought it would make sense for it to be part of `GENERATED_FILES`. But I don't think it changes much in practice besides feeling more natural to me, so I'm marking this as resolved.